src/Console/Traits/HandlesCleanup.php

```
@@ -43,7 +43,7 @@ public function getSubscribedSignals(): array
      *
      * @return int|false The exit code to return or false to continue the normal execution
      */
-    public function handleSignal(int $signal, /* int|false $previousExitCode = 0 */)
+    public function handleSignal(int $signal, int|false $previousExitCode = 0): int|false
```

PHPMD (2.15.0)

Avoid unused parameters such as '$previousExitCode'. (undefined)

(UnusedFormalParameter)